src/zmq/zmqconfig.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_ZMQGLOBAL_H
```

Comment:
This needs to be ``BITCOIN_ZMQ_ZMQCONFIG_H`` also don't forget the header end comment.